src/resources/postcss/components/_navigation.pcss

```diff
@@ -0,0 +1,93 @@
+/* = Navigation
+============================ */
+.tribe-events {
+
+	.tribe-common-c-nav {
+		display: flex;
+		justify-content: space-between;
+		border-top: 1px solid var(--color-border-default);
+		padding-top: var(--spacer-6);
+	}
+
+	.tribe-common-c-nav--prev,
+	.tribe-common-c-nav--next {
+		display: flex;
+		align-items: center;
+		color: var(--color-text-secondary);
+		font-size: var(--font-size-1);
+		font-weight: var(--font-weight-bold);
+
+		@media (--viewport-medium) {
+			font-size: var(--font-size-2)
+		}
+
+		&:hover {
+			color: rgba(var(--color-text-primary), 0.8);
+		}
+
+		&:disabled {
+			color: var(--color-text-disabled);
+		}
+	}
+
+	.tribe-common-c-nav--prev {
+
+		&:before {
+			background-image: svg-inline(caret-left);
+			background-repeat: no-repeat;
+			background-size: contain;
+			content: '';
+			height: 14px;
+			margin-right: 10px;
+			width: 12px;
```

Comment:
let's move `background-repeat`, `background-size`, `content`, `height`, and `width` up to the block with both `prev` and `next` (keep things DRY)